docs/components/modeler/bpmn/timer-events/timer-events.md

```diff
@@ -79,11 +79,13 @@ If the duration is zero or negative, the timer fires immediately.
 
 A cycle defined as ISO 8601 repeating intervals format; it contains the duration and the number of repetitions. If the repetitions are not defined, the timer repeats infinitely until it is canceled.
 It's possible to define a start time. By doing this, the timer triggers for the first time on the given start time. Afterwards, it will follow the interval as usual.
+Additionally, you can specify a time cycle using cron expressions. Please see the [CronExpression Tutorial](https://spring.io/blog/2020/11/10/new-in-spring-5-3-improved-cron-expressions) for additional information about using cron expressions.
```

Comment:
You can remove "please" in our docs for conciseness. Simply "See the [CronExpression Tutorial](https://spring.io/blog/2020/11/10/new-in-spring-5-3-improved-cron-expressions) for additional information about using cron expressions." will do.